### PR TITLE
Custom style css

### PR DESCRIPTION
**Pull request by darnault**
This branch has objective to lay the foundations for the websites custom stylesheet, reducing the dependence on bootstrap files. Future plans include implementing personalized grid system with responsive web techniques.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h4 class="modal-title">Credits</h4>
 			</div>
 		</div> <!-- END MODAL -->
 
-		<div class="container">
+		<div class="myGrid">
```
Comment (iocedano) [inline index.html:43]: There are some of styles naming convertions  for css just to let you know https://css-tricks.com/css-style-guides/

**Comment by iocedano** [comment]
Maybe you like to make a rebase

**Comment by darnault** [comment]
I was thinking of that.
But, why would it be better then simple merge?

**Comment by darnault** [comment]
As soon as I get home.

**Pull request closed without merging**